website/docs/components/copy/snippet/partials/code/component-api.md

```diff
@@ -0,0 +1,22 @@
+## Component API
+
+This component uses [ember-cli-clipboard](https://github.com/jkusa/ember-cli-clipboard) under the hood.
+
+<Doc::ComponentApi as |C|>
```

Comment:
Still to decide/understand if we want to expose the `@encoded` argument (see comment in the other PR)